miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultStorageLocationService.java

```diff
@@ -110,15 +125,27 @@ private long update(StorageLocation freezer) {
   private void validateChange(StorageLocation storage, StorageLocation beforeChange) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "beforeChange". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)